modules/openapi-generator/src/main/resources/python/asyncio/rest.mustache

```diff
@@ -65,16 +65,13 @@ class RESTClientObject(object):
             ssl=ssl_context
         )
 
+        self.proxy = configuration.proxy
+        self.proxy_headers = configuration.proxy_headers
+
         # https pool manager
-        if configuration.proxy:
-            self.pool_manager = aiohttp.ClientSession(
-                connector=connector,
-                proxy=configuration.proxy
-            )
```

Comment:
This code was added 3 years ago so it looks like the signature has changed in that time.